sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatterSuite.scala

```diff
@@ -76,13 +76,49 @@ class CodeFormatterSuite extends SparkFunSuite {
 
     val reducedCode = CodeFormatter.stripExtraNewLinesAndComments(code)
     assert(reducedCode ===
-      """
-        |public function() {
+      """public function() {
         |code_body
         |code_body
         |code_body
-        |}
-      """.stripMargin)
+        |}""".stripMargin)
+  }
+
+  test("removing extra new lines and comments 2") {
```

Comment:
How about just using the title here, `improves extra new line and comment removals`?

Reply:
ok, renamed